### PR TITLE
Release aws package with `related.entity` (and add missing data)

### PR DESCRIPTION
**Pull request by romulets**
Release aws package (2.26.0) with the new `related.entity` field 

Shipping together the addition of the field values to `related.entity`:

- `json.userIdentity.accessKeyId`
- `json.requestParameters.serialNumber`
- `json.responseElements.user.userId`
- `json.responseElements.serialNumber`

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [x] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices)

### CODE REVIEW

**Review by andrewkroh — APPROVED** [review]

**Review by efd6 — APPROVED** [review]

**Review by constanca-m — APPROVED** [review]

File: packages/aws/manifest.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ conditions:
   elastic:
     subscription: basic
   kibana:
-    version: "^8.16.0"
+    version: "^8.15.0"
```
Comment (gpop63) [inline packages/aws/manifest.yml:13]: Why was the kibana version downgraded here?

Comment (romulets) [reply packages/aws/manifest.yml:13]: It was upgraded on my initial PR, which in forces any new release to be backported. The goal is to unblock other people to push code to main and release versions

Comment (kaiyan-sheng) [reply packages/aws/manifest.yml:13]: @romulets thanks!!

**Review by kaiyan-sheng — APPROVED** [review]

**Review by gpop63 — APPROVED** [review]

**Review by agithomas — APPROVED** [review]
LGTM!

**Pull request merged**